src/util-inl.h

```diff
@@ -295,6 +295,14 @@ std::string ToUpper(const std::string& in) {
   return out;
 }
 
+// TODO(@juanarbol): change the name of this.
+inline uint32_t cares_get_32bit(const unsigned char* p) {
```

Comment:
To be honest, I don't have a clue about what is this function actually doing (and don't know how to name it by extension)

Reply:
I think a generic way to name it might be `ReadUint32BE`, analogous to the JS [`Buffer.readUint32BE` function](https://nodejs.org/api/buffer.html#buffer_buf_readuint32be_offset) which does the same thing? Saying that it reads a number at a specific memory address as a 32-bit big endian integer is definitely accurate here :)

Reply:
Oh, by the way, there should be a declaration of this function in `util.h` as well – that may or may not also already solve your problem with `base64.h`. The declaration should have the `inline` tag, but the definition here doesn’t need that anymore then – basically, the same thing as we do for `StringEqualNoCase()` right below :slightly_smiling_face: